pkg/services/auth/auth_token_test.go

```diff
@@ -46,6 +49,23 @@ func TestUserAuthToken(t *testing.T) {
 				So(err, ShouldEqual, ErrAuthTokenNotFound)
 				So(LookupToken, ShouldBeNil)
 			})
+
+			Convey("signing out should delete token and cookie if present", func() {
+				token, err := userAuthTokenService.CreateToken(userID, "192.168.1.1:1234", "user agent")
+				So(err, ShouldBeNil)
+				So(token, ShouldNotBeNil)
+
+				httpreq := &http.Request{Header: make(http.Header)}
+				httpreq.AddCookie(&http.Cookie{Name: userAuthTokenService.Cfg.LoginCookieName, Value: token.AuthToken})
```

Comment:
Probably need to use unhashed token as cookie value to resolve failing test.

Reply:
🤦‍♂️